spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe 'user#show', type: :feature do
+  describe 'show user detail page' do
+    before(:each) do
+      @user1 = User.create(name: 'Aisha', photo: 'https://unsplash.com/photos/7mBictB_urk', bio: 'Accountant',
+                           postscounter: 0)
+      @user2 = User.create(name: 'Moses', photo: 'https://unsplash.com/photos/7mBictB_urk', bio: 'Engineer',
+                           postscounter: 2)
+      @post1 = Post.create(title: 'This is a post', text: 'Accounting made easy', likes_counter: 0,
+                           comments_counter: 0, author_id: @user1.id)
+      @post2 = Post.create(title: 'Engineering post', text: 'How to become a mechanical eng', likes_counter: 0,
+                           comments_counter: 0, author_id: @user2.id)
+      @post3 = Post.create(title: 'This is another post', text: 'Benefits of budgeting', likes_counter: 0,
+                           comments_counter: 0, author_id: @user1.id)
+      @post4 = Post.create(title: 'Tutorial', text: 'Accounting made easy', likes_counter: 0,
+                           comments_counter: 0, author_id: @user1.id)
+      @post5 = Post.create(title: 'Automobile engineering', text: 'Reading', likes_counter: 0,
+                           comments_counter: 0, author_id: @user2.id)
+      @comment1 = Comment.create(text: 'This is great', post_id: @post1.id, author_id: @user2.id)
+      visit user_path(@user1)
+    end
+
```

Comment:
* As mentioned in the requirement your tests should cover the user index, show actions and post index, show actions 
* Kindly separate the tests into four different files and cover all [cases mentioned in the project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20user%27s,for%20any%20other%20features%20you)